src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -30,6 +30,7 @@ class PressableWithSecondaryInteraction extends Component {
      */
     executeSecondaryInteractionOnContextMenu(e) {
         const selection = window.getSelection().toString();
+        e.stopPropagation();
```

Comment:
Hi, why do we need this line? It seems to be unnecessary. cc @roryabraham @parasharrajat @puneetlath @Drewfergusson